homeassistant/components/zwave/__init__.py

```diff
@@ -769,6 +770,7 @@ def __init__(self, value, domain):
         self._wakeup_value_id = None
         self._battery_value_id = None
         self._power_value_id = None
+        self._scheduled_update = False
```

Comment:
For a future PR, maybe rename this to `_update_scheduled` to avoid mixup with `_schedule_update` method.